applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h

```diff
@@ -236,9 +259,10 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientThermalElement : public Ele
         const Vector&                                    rIntegrationCoefficients,
         const ProcessInfo&                               rCurrentProcessInfo) const
     {
-        GeoThermalDispersionLaw law{TDim};
-        const auto              constitutive_matrix =
-            law.CalculateThermalDispersionMatrix(GetProperties(), rCurrentProcessInfo);
+        std::size_t NumberOfDimensions = GetGeometry().LocalSpaceDimension();
```

Comment:
Can be const and since it's a local variable, it should be snake case
```suggestion
        const std::size_t number_of_dimensions = GetGeometry().LocalSpaceDimension();
```

Reply:
I think you may have forgotten to change the variable name to snake case...?